packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -74,17 +71,7 @@ function MaxButton(props: React.ButtonHTMLAttributes<HTMLButtonElement>) {
 function TransferPanelInputField(
   props: React.InputHTMLAttributes<HTMLInputElement>
 ) {
-  const { value = '', onChange, ...rest } = props
-  const setAmount = useSetInputAmount()
-  const decimals = useSelectedTokenDecimals()
-
-  useEffect(() => {
```

Comment:
This seems to be already sanitized without this part

Reply:
nice kill